config.yaml

```diff
@@ -27,6 +29,10 @@ contracts:
       - event: Mint(address sender, address indexed owner, int24 indexed tickLower, int24 indexed tickUpper, uint128 amount, uint256 amount0, uint256 amount1)
       - event: SetFeeProtocol(uint8 feeProtocol0Old, uint8 feeProtocol1Old, uint8 feeProtocol0New, uint8 feeProtocol1New)
       - event: Swap(address indexed sender, address indexed recipient, int256 amount0, int256 amount1, uint160 sqrtPriceX96, uint128 liquidity, int24 tick)
+  - name: NFPM 
+    handler: src/EventHandlers/NFPM.ts
+    events:
+      - event: Transfer(address indexed from, address indexed to, uint256 indexed tokenId)
```

Comment:
Main new addition. I'm adding tracking of the NFT CL positions here along with tracking their Transfer events.